src/database/Persona/Persona.db.ts

```diff
@@ -293,6 +293,44 @@ export async function queryProfilesDB(
     return result
 }
 
+export async function queryProfilesPagedDB(
+    options: {
+        after: ProfileIdentifier | undefined
+        query?: string
+    },
+    count: number,
+): Promise<ProfileRecord[]> {
+    const t = createTransaction(await db(), 'readonly')('profiles')
+    const breakPoint = options.after?.toText()
+    let firstRecord = true
+    const data: ProfileRecord[] = []
+    for await (const rec of t.objectStore('profiles').iterate()) {
+        if (firstRecord && breakPoint && rec.key !== breakPoint) {
+            rec.continue(breakPoint)
+            firstRecord = false
+            continue
+        }
+        firstRecord = false
+        if (rec.key === breakPoint) {
+            // after this record
+            continue
+        }
+        if (count <= 0) break
```

Comment:
This condition could be moved to the outside of iteration.

Reply:
I'm doing `count--` in the iteration